libs/domains/application/src/lib/slices/applications.slice.ts

```diff
@@ -92,13 +92,13 @@ export const fetchApplicationCommits = createAsyncThunk<Commit[], { applicationI
   }
 )
 
-export const fetchApplicationDeployments = createAsyncThunk<DeploymentHistoryApplication[], { applicationId: string }>(
-  'application/deployments',
-  async (data) => {
-    const response = await applicationDeploymentsApi.listApplicationDeploymentHistory(data.applicationId)
-    return response.data.results as DeploymentHistoryApplication[]
-  }
-)
+export const fetchApplicationDeployments = createAsyncThunk<
+  DeploymentHistoryApplication[],
+  { applicationId: string; silently: boolean }
```

Comment:
silently? to make it optional. And if you can set a default value to false that's cool